src/main/java/net/minecraftforge/resource/ResourceCacheManager.java

```diff
@@ -40,8 +40,12 @@
  * <p>
  * This class handles caching the resource listing calls on a pack, pack type and namespace level.
  */
-public class ResourceCacheManager
-{
+@ApiStatus.Internal
+public class ResourceCacheManager {
+    /**
+     * Logging marker to handle the hiding of the entries from the debug file.
+     */
+    private static final Marker RESOURCE_CACHE_MARKER = MarkerFactory.getMarker("RESOURCE-CACHE");
```

Comment:
The marker does not need to be suffixed with `_MARKER`, as is done for all other markers in Forge code.
```suggestion
    private static final Marker RESOURCE_CACHE = MarkerFactory.getMarker("RESOURCE-CACHE");
```